lib/streamlit/runtime/caching/cache_utils.py

```diff
@@ -91,7 +92,7 @@ class ElementMsgData:
     id_of_dg_called_on: str
     returned_dgs_id: str
     widget_metadata: Optional[WidgetMsgMetadata] = None
-    media_data: Optional[List[MediaMsgData]] = None
+    media_data: Optional[list[MediaMsgData]] = None
```

Comment:
You can also get rid of these `Optional`s by use of the union operator. I think `list[MediaMsgData] | None` looks better, at least.